packages/metadata-fields/BasicMetadata.jsx

```diff
@@ -190,13 +190,14 @@ export default function BasicMetadata({ data }) {
         ""
       )}
 
+      {/* TODO: there are 2 leading classes here - do we actually want 125% or 26.25px? Same for line 222 */}
       {campus &&
       !["undefined", undefined, null, ""].includes(campus.asset_assetid) ? (
         <div className="su-border-t border-t-black-20">
           <BasicFields title="Campus unit">
             <a
               href={campus.asset_url}
-              className="su-no-underline su-leading-[22.5px] hover:su-underline su-text-digital-red dark:su-text-dark-mode-red dark:hover:su-text-dark-mode-red hover:su-text-digital-red su-text-18 su-leading-[26.25px]"
+              className="su-no-underline su-leading-[125%] hocus:su-underline su-text-digital-red dark:su-text-dark-mode-red dark:hocus:su-text-dark-mode-red hocus:su-text-digital-red su-text-18 su-leading-[26.25px]"
```

Comment:
Something to double check which leading class you'd like to use here - added a TODO